components/skills.tsx

```diff
@@ -0,0 +1,19 @@
+import { CertificatesProps } from "@/types/certificates"
+
+export default function Skills({ certificates }: CertificatesProps) {
+    const skills = [...new Set(certificates
+        .map((item) => item.skills)
+        .flat())]
+    
+    return (
+        <section className='m-8 flex flex-col gap-4 justify-center items-center'>
+            <h2 className='text-xl font-bold text-slate-800'>Tech stack</h2>
+            <div className='flex gap-2 m-2'>
+                {skills.map((item: string) =>
```

Comment:
Consider use `forEach` method of the `Set` class

Reply:
I've tried that, but because forEach does not return JSX, I can't just replace the map() method with it. I believe I would have to build an array of JSX elements by using forEach to push items into an array outside of return() first. In this case, it seems to me more straightforward to turn the Set into an array. What do you think?